### ISSUE TITLE
Event Binding In Page Element not Working

### ISSUE THREAD

**Issue by GHR-Arash** [blasten/turn.js#432]
cant bind any event to element in page of book 
below i am trying to bind click event to sampelSpan , as sample , but it doesn't work 

```
$(document).ready(function(e) {
$("#book").turn("page",$("#book").turn("pages"));
$("#sampelSpan").click(function(e) {

alert('some text');
});

});


<div id="book" >

<div class="even" ><div class="data">1</div> </div> 
<div class="odd"> <div class="data">2</div> </div>
<div class="even" ><div class="data">3</div> </div> 
<div class="odd"> <div class="data"><span id="sampelSpan">Hello TurnJs</span></div></div>

</div>
```

**Comment by Screenfeed** [comment]
Same problem here. All events are killed because the plugin removes some pages from the dom and add them back later... without their attached events.
So any lightbox/player/whatever won't work >_<

**Comment by Screenfeed** [comment]
Busted \o/

In the plugin, replace all `.remove()` with `.detach()` (it's the same thing but it keeps attached events).
The only `.remove()` you can keep are the ones for `fparent.remove()` and `shadow.remove()`.
Not fully tested but it does the trick so far.
